torch/testing/_internal/common_methods_invocations.py

```diff
@@ -400,6 +400,67 @@ def __init__(self,
         #   outside a function's domain.
         self._domain_eps = 1e-5
 
+
+def sample_inputs_binary(op_info, device, dtype, requires_grad):
```

Comment:
See https://github.com/pytorch/pytorch/pull/56227, where @imaginary-person is added sample inputs for binary pwise ops, too.

Maybe we can take the union of the both of them?

Reply:
They are already. The only thing missing here is the explicit use of a python scalar instead of a scalar tensor.